lib/sparse/src/index/inverted_index/mod.rs

```diff
@@ -32,4 +32,7 @@ pub trait InvertedIndex {
     ) -> std::io::Result<Self>
     where
         Self: Sized;
+
+    /// Number of indexed vectors
+    fn indexed_vector_count(&self) -> usize;
```

Comment:
I guess we can rename it into `vector_count`. we already know here that vectors are indexed